Description: Use 'less -R' as default pager 
 Some users had issues when diff produces colors, so make it easier
 for them.
Author: Martin Quinson <mquinson@debian.org>
Last-Update: 2012-12-27
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/quilt/+bug/933617
Upstream-status: to be submitted 

Index: quilt.git/quilt/scripts/patchfns.in
===================================================================
--- quilt.git.orig/quilt/scripts/patchfns.in	2012-12-27 16:28:38.118656473 +0100
+++ quilt.git/quilt/scripts/patchfns.in	2012-12-27 16:28:38.286659096 +0100
@@ -999,9 +999,9 @@
 {
 	test -t 1 || return 0
 
-	# QUILT_PAGER = QUILT_PAGER | GIT_PAGER | PAGER | less
+	# QUILT_PAGER = QUILT_PAGER | GIT_PAGER | PAGER | less -R
 	# NOTE: QUILT_PAGER='' is significant
-	QUILT_PAGER=${QUILT_PAGER-${GIT_PAGER-${PAGER-less}}}
+	QUILT_PAGER=${QUILT_PAGER-${GIT_PAGER-${PAGER-less -R}}}
 
 	[ -z "$QUILT_PAGER" -o "$QUILT_PAGER" = "cat" ]  && return 0
 
Index: quilt.git/doc/quilt.1.in
===================================================================
--- quilt.git.orig/doc/quilt.1.in	2012-12-27 16:28:38.250658534 +0100
+++ quilt.git/doc/quilt.1.in	2012-12-27 16:28:51.838870743 +0100
@@ -213,7 +213,7 @@
 
 The pager quilt shall use for commands which produce paginated output. If
 unset, the values of GIT_PAGER or PAGER is used.  If none of these variables
-is set, "less" is used.  An empty value indicates that no pager should be
+is set, "less -R" is used.  An empty value indicates that no pager should be
 used.
 
 .IP EDITOR 4
